config/quickwit.yaml

```diff
@@ -15,9 +15,9 @@ version: 0
 #
 # Path to directory where data is persisted. Default to `./qwdata`.
 #
-#data_dir: /path/to/data
+#data_dir_path: /path/to/data
```

Comment:
I think we should keep here `data_dir` to remain consistent with the env variable too. We need to modify `QuickwitConfig` so that it can parse `data_dir`

Reply:
ok